test/verify/check-machines-dbus

```diff
@@ -175,6 +175,46 @@ class TestMachinesDBus(machineslib.TestMachines):
         b.wait_in_text("#vm-subVmTest1-network-1-source", "test_network")
         b.wait_not_present("#vm-subVmTest1-network-1-source-tooltip")
 
+    def testChangeMemory(self):
+        b = self.browser
+        m = self.machine
+
+        self.startVm("subVmTest1")
+
+        self.login_and_go("/machines")
+        b.wait_in_text("body", "Virtual Machines")
+        b.wait_present("tbody tr th")
+        b.wait_in_text("tbody tr th", "subVmTest1")
+
+        b.click("tbody tr th") # click on the row header
+        b.wait_present("#vm-subVmTest1-state")
+        b.wait_in_text("#vm-subVmTest1-state", "running")
+
+        b.wait_present("#vm-subVmTest1-overview") # wait for the tab
+        b.click("#vm-subVmTest1-overview") # open the "Overview" subtab
+
+        # Wait for the edit link
+        b.wait_present("#vm-subVmTest1-memory-modal-memory-count")
+        b.click("#vm-subVmTest1-memory-modal-memory-count")
+
+        # Make sure the footer warning does not appear until we change something
```

Comment:
Please create test that you check that footer warning is present when you change max memory, not only not present when you don't change.

Reply:
Right now, footer warning never shows. I made sure max memory input is greyed out when vm is running, so there is no way you i can make footer warning show. But i don't know if designers will like that approach. If they will approve, a can delete the footer warning part and therefore there will be no need for testing it.

Reply:
Makes sense.